roles/openshift_node_upgrade/tasks/restart.yml

```diff
@@ -22,7 +22,6 @@
     - "{{ openshift.common.service_type }}-master-controllers"
     - "{{ openshift.common.service_type }}-node"
   failed_when: false
-  when: openshift.common.is_containerized | bool
```

Comment:
Ideally we should not be restarting master services unnecessarily, for rpm based installations here they should be independent of docker afaik, so we should leave them running if at all possible.